numpy/add_newdocs.py

```diff
@@ -463,6 +463,45 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Convenience function to create multiple nditers from a single ndarray.
+
+    Parameters (see nditer)
+    ---------
+    op : ndarray
+        The array to iterate over.
+
+    axes :  sequence of sequences
+        As in nditer "op_axes"
```

Comment:
How about `Each item is used as an "op_axes" argument to an nditer` or something to that effect.